test/distributed/org/apache/cassandra/distributed/test/GossipTest.java

```diff
@@ -46,13 +47,13 @@
 import org.apache.cassandra.streaming.StreamPlan;
 import org.apache.cassandra.streaming.StreamResultFuture;
 import org.apache.cassandra.utils.FBUtilities;
+import org.assertj.core.api.Assertions;
```

Comment:
nit: import ordering

Reply:
I asked IntelliJ to fix and it didn't do anything... what is the expected order?

There is a block above with junit, do you want this there?  This class doesn't look consistent =(

Reply:
I _think_ the junit stuff goes above o.a.c. See Imports here: https://cassandra.apache.org/_/development/code_style.html